shell/conf/hdinsight_defaults.conf

```diff
@@ -22,7 +22,7 @@ CLUSTER_DISKS="RR1"
 #Cost per remote/hour $0.024/GB/month USD 24$ /month Locally Redundant
 clusterCostDisk="0.17" # 5TB/hr
 
-resourceGroup="Default-Storage-SouthCentralUS"
+resourceGroup="alojadlapremium"
```

Comment:
Maybe it's right (I have no idea), but this looks like a typo since I would expect `aloja` `adla`.

Reply:
A resource group is a set of resources in Azure so we can easily manage them. The only one that was available was the one you see. It makes no difference at all and arranges all our clusters, both HDI and adla together for easier management

Reply:
Great, but all I'm saying is that `alojadlapremium` looks wrong, because `aloja` ends in an `a` and `adla` starts in an `a`, but there is only one `a` between them. Makes sense?

Since we are still reviewing config changes (did I mention that these have no place in a public repo?), I thought that I could at least try to help by pointing out a possible typo in the name of the resource group. If it's definitely right, then nothing to worry about.

Reply:
In our case "DLA" makes complete sense:

dla = data lake analytics
adla = azure data lake analytics

The command line of azure uses "az dla" y "az dls" not adla. No solo eso si no que una de nuestras cuentas es alojadw not alojaadw (Data Warehouse)

In essence, I wouldn't say it's a typo.